src/test/unit_tests/jest_tests/test_nc_nsfs_account_cli.test.js

```diff
@@ -103,6 +103,24 @@ describe('manage nsfs cli account flow', () => {
             assert_account(account_symlink, account_options);
         });
 
+        it('should fail - cli create account invalid access_key (leading space in access_key)', async () => {
+            const { type, name, uid, gid, access_key, secret_key } = defaults;
+            const account_options = { config_root, name, uid, gid, secret_key};
```

Comment:
I would add a short comment that you didn't pass the `access_key` intentionally.